app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -463,6 +474,15 @@ public boolean onLongClick(final View v) {
             case R.id.detail_controls_download:
                 NavigationHelper.openDownloads(getActivity());
                 break;
+
+            case R.id.detail_uploader_root_layout:
+                if (TextUtils.isEmpty(currentInfo.getSubChannelUrl())) {
```

Comment:
Why is this checking whether a subchannel exists, but then the parent uploader url is used?

Reply:
We open the sub-channel when it exists on click. On long-click we want to open the parent channel, but only if it has a sub-channel, as you could just click when it doesn't have a sub-channel to open the parent channel.

Reply:
Oh, I see, sorry